eos/maths/angular-integrals.cc

```diff
@@ -0,0 +1,102 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2025 Florian Herren
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/maths/angular-integrals.hh>
+#include <eos/maths/power-of.hh>
+
+#include <algorithm>
+#include <cmath>
+
+namespace eos
+{
+    double wigner_3j(const int & j1, const int & m1, const int & j2, const int & m2, const int & j3, const int & m3)
+    {
+        if (m1 + m2 + m3 != 0 || j1 < 0 || j2 < 0 || j3 < 0) return 0.0;
+
+        double ret = 0.0;
+        const double pref = (((j1 - j2 - m3) % 2) ? -1.0 : 1.0)
+                    * std::sqrt(std::tgamma(j1 + j2 - j3 + 1.0)) * std::sqrt(std::tgamma(j1 - j2 + j3 + 1.0))
```

Comment:
Please use integer literals in the arguments of ``std::tgamma`` since it is overloaded for integers.